molecule/java8/molecule.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+driver:
+  name: docker
+
+platforms:
+  - name: sansible_java-trusty
+    image: ubuntu:trusty
+  - name: sansible_java-xenial
+    image: ubuntu:xenial
+  - name: sansible_java-bionic
+    image: ubuntu:bionic
+
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
```

Comment:
We don't want this turned on, no more merge in Sansible roles :)

Reply:
good spot - I've just used previous version.